skymp5-front/src/constructorComponents/chat/dices/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useRef } from 'react';
+import './styles.scss';
+import D100 from './icons/D100';
+import Coin from './icons/Coin';
+import D6 from './icons/D6';
+import D12 from './icons/D12';
+import D20 from './icons/D20';
+import Pouch from './icons/Pouch';
+
+const Dices = (props: {
+  send: (msg: string) => void,
+  disableSound: boolean,
+  isOpened: boolean,
+  setOpened: (boolean) => void,
+}) => {
+  const count = useRef(0);
+
+  const roll = (type: 'coin' | 'dice', code: string) => {
+    if (count.current < 10) {
+      if (!props.disableSound) { playSound(type); }
+      props.send(`/${code}`);
+      count.current += 1;
+    }
+  };
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      count.current = 0;
+    }, 1000 * 60);
+    return () => clearInterval(interval);
+  }, []);
+
+  const playSound = (type: 'pouch' | 'coin' | 'dice') => {
+    if (type === 'dice') {
+      const rand = Math.floor(Math.random() * 3) + 1;
+      const audio = new Audio(require(`../../../sound/dice${rand}.mp3`).default);
```

Comment:
Please use import over require.

Reply:
decided not to change, because we use dynamic import path